CMakeLists.txt

```diff
@@ -113,14 +113,13 @@ if (MPI)
     file(GLOB parpacksrc_STAT_SRCS PARPACK/SRC/MPI/*.f)
 endif()
 
-#add_library(arpack SHARED
-#                   ${arpackblas_STAT_SRCS}
-#                   ${arpacklapack_STAT_SRCS}
-#                   ${arpackutil_STAT_SRCS}
-#                   ${arpacksrc_STAT_SRCS})
-add_library(arpack ${arpackutil_STAT_SRCS}
-                   ${arpacksrc_STAT_SRCS}
-                   ${arpacksrc_ICB})
+option(ENABLE_STATIC "Static linking (shared otherwise)" ON)
```

Comment:
@sylvestre @fghoussen 

What's the rational in adding this option? [add_library](https://cmake.org/cmake/help/v3.0/command/add_library.html) already support the default `-D BUILD_SHARED_LIBS `:

> If no type is given explicitly the type is STATIC or SHARED based on whether the current value of the variable BUILD_SHARED_LIBS is ON.

so IMO this PR should be reverted. This would also avoid code duplication @sylvestre mentioned below.

Reply:
see https://github.com/opencollab/arpack-ng/pull/94